skfda/exploratory/visualization/_phase_plane_plot.py

```diff
@@ -0,0 +1,99 @@
+"""Phase-Plane Plot Module.
+
+This module contains the functionality in charge of plotting
+two different functions as coordinates, this can be done giving
+one FData, with domain 1 and codomain 2, or giving two FData, both
+of them with domain 1 and codomain 1.
+"""
+
+from typing import Any, Optional, Union
+
+from matplotlib.axes import Axes
+from matplotlib.figure import Figure
+
+from ...representation import FData
+from ._utils import _get_figure_and_axes, _set_figure_layout
+
+
+class PhasePlanePlot:
```

Comment:
As this is more generic than a phase plane plot (which could be a particular subclass of this one), I propose renaming this to `ParametricPlot` which is more clear (and it also opens the door to extend this plot in the third dimension, as it makes no references to a "plane").

Reply:
ok, good idea